changelog.md

```diff
@@ -1,4 +1,32 @@
 
+## 3.0.0 / 2020-08-XX
+
+### New Features
+
+- Tracim Live Message is now available. Its now not necessary to refresh Tracim web page manually, all new contents information are automatically available in your web browser
+- Tracim permit connection with a username instead of email. This username feature is mandatory to introduce mention feature (available in next release)
+
+### Fixed Issues
+
+- Frontend: #2615, #2747, #2783, #2986, #2999, #3096, #3135, #3225, #3236, #3239, #3242, #3255, #3257, #3274, #3289, #3295, #3303, #3312, #3393
+- Backend: #2939, #2970, #2979, #3025, #3050, #3064, #3091, #3119, #3370, #3373
+- Docker: #3222, #3383
+- Documentation: #3154
+
+### Other Changes
+
+- Tracim frontend use Yarn now instead of npm
```

Comment:
```suggestion
- Tracim frontend is now built with Yarn instead of NPM. See how to migrate your working copy of Tracim here: https://github.com/tracim/tracim/issues/2916
```